sdk/core/System.ServiceModel.Rest/src/ResponseErrorClassifier.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+
+namespace System.ServiceModel.Rest.Core
+{
+    /// <summary>
+    /// TBD.
+    /// </summary>
+    public class ResponseErrorClassifier
```

Comment:
Our base ResponseClassifier in Azure.Core has methods for both errors and retries.  I like this split with a type that does errors only, but not sure how far we can push it to maintain the public APIs we have in Core today.